adrs/2023-02-02-automate-runner-updates.md

```diff
@@ -0,0 +1,26 @@
+# Automate updating runner version
+
+**Status**: Proposed
+
+## Context
+
+When a new [runner](https://github.com/actions/runner) version is released, new
+images need to be built in
+[actions-runner-controller/releases](https://github.com/actions-runner-controller/releases).
+This is currently started by the
+[release-runners](https://github.com/actions/actions-runner-controller/blob/master/.github/workflows/release-runners.yaml)
+workflow, although this only starts when the set of file containing the runner
+version is updated (and this is currently done manually).
+
+## Decision
+
+We can have another workflow running on a cadence (hourly seems sensible) and checking for new runner
+releases, creating a PR with the updated version if there is one. Once that PR
+is merged, the existing workflow will pick things up.
```

Comment:
Could we also add the alternatives and why we dismissed them?

Reply:
On it, thanks 👍